ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/events/EventsTableTreeEvents.scala

```diff
@@ -86,31 +86,6 @@ private[events] trait EventsTableTreeEvents { this: EventsTable =>
     "exercise_child_event_ids",
   ).mkString(", ")
 
-  private val groupByColumns = Seq(
-    "event_offset",
-    "transaction_id",
-    "node_index",
-    "participant_events.event_id",
-    "contract_id",
-    "ledger_effective_time",
-    "template_id",
-    "command_id",
-    "workflow_id",
-    "application_id",
-    "submitter",
-    "create_argument",
-    "create_signatories",
-    "create_observers",
-    "create_agreement_text",
-    "create_key_value",
-    "exercise_consuming",
-    "exercise_choice",
-    "exercise_argument",
-    "exercise_result",
-    "exercise_actors",
-    "exercise_child_event_ids",
-  ).mkString(", ")
```

Comment:
Good catch removing the `group by`s, they should have been removed since we had witnesses directly in the events table, too bad it slipped so far.